src/NServiceBus.AcceptanceTests/Routing/When_using_custom_routing_strategy.cs

```diff
@@ -25,16 +25,16 @@ public class When_using_custom_routing_strategy : NServiceBusAcceptanceTest
                         await session.Send(new MyCommand { Instance = Discriminator1 });
                         await session.Send(new MyCommand { Instance = Discriminator2 });
                         await session.Send(new MyCommand { Instance = Discriminator1 });
-                        await session.Send(new MyCommand { Instance = Discriminator2 });
+                        await session.Send(new MyCommand { Instance = Discriminator1 });
```

Comment:
With equal number of messages for both `1` and `2` there is a chance that the test could pass if round-robin distribution occurred and distribution did not. It's not a big chance, but it was one that we though could provide a false positive on this code.